packages/select-rich/index.js

```diff
@@ -1,10 +1,11 @@
 /**
- * @deprecated
+ * !deprecated
```

Comment:
Latest typescript is complaining about this for some dumb reason so I just prefix it with something else.. just FYI in case you're searching for these.